scripts/linters/python_linter.py

```diff
@@ -320,6 +345,40 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from pylint error messages.
+
+        Args:
+            lint_messages: list(str). Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = []
+        # Remove newlines and coverage report from the end of message.
+        # we need to remove last five items from the list because starting from
+        # end we have three lines with empty string(''), fourth line
+        # contains the coverage report i.e.
+        # Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
+        # and one line with dashes(---).
+        newlines_present = lint_messages[-1] == '\n' and (
+            lint_messages[-2] == '\n' and lint_messages[-4] == '\n')
+        coverage_present = re.search(
+            r'previous run: \d+.\d+/\d+,', lint_messages[-3])
+        dashes_present = re.search(r'-+', lint_messages[-5])
+        if newlines_present and coverage_present and dashes_present:
+            lint_messages = lint_messages[:-5]
+        for message in lint_messages:
+            # Every pylint message has a message id inside the brackets
+            # we are removing them here.
+            if message.endswith(')'):
+                message_id = re.search(r'\((\w+-*)+\)$', message).group(0)
+                error_messages.append(message.replace(message_id, ''))
```

Comment:
Same thing, make sure to remove only the last occurrence.

Reply:
Done.